bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/SMSService.java

```diff
@@ -0,0 +1,12 @@
+package org.bahmni.module.bahmnicore.service;
+
+import org.openmrs.Patient;
+
+import java.util.Date;
+
+public interface SMSService {
+
+    String getPrescriptionMessage(Object[] prescriptionArguments);
```

Comment:
should not take in object array. Use typed parameters

Reply:
refactored to typed parameters